packages/pipeline/src/entities/token_order.ts

```diff
@@ -6,22 +6,24 @@ import { bigNumberTransformer, numberToBigIntTransformer } from '../utils';
 
 @Entity({ name: 'token_orderbook_snapshots', schema: 'raw' })
 export class TokenOrderbookSnapshot {
-    @PrimaryColumn({ name: 'source' })
+    @Column({ name: 'source' })
```

Comment:
I think we still need `source` to be a primary column because we could theoretically have two different snapshots from different sources at the same time.